File: exer14.c
/**
 * Exercício 14, lista de exercícios 06-03-2013
 * Programa para converter uma quantidade de segundos em horas, minutos e segundos
 */

/**
 * defindo constante de conversão
 */
#define FATOR 60

#include <stdio.h>

int main()
{

	int seg, horas, min;

	printf("%s\n", "Converta segundos para horas, minutos e segundos");
	printf("%s\n", "Digite a quantidade de segundos para a conversão:");
	scanf("%d", &seg);

	horas = seg / (FATOR * FATOR);
	min = (seg % (FATOR * FATOR)) / FATOR;
	seg = seg % FATOR;

	printf("%s: %d:%d:%d\n", "Os segundos convertidos ficam", horas, min, seg);

	return 0;
}

File: exer14.pas
Program CoverterSegundos ;

Const
	FATOR = 60;
	
Var
	seg, min, horas: integer;

Begin

	writeln('Converta segundos para horas, minutos e segundos!');
	writeln('Digite a quantidade de segundos para a conversão:');
	readln(seg);
	
	horas := seg div (FATOR * FATOR);
	min := (seg mod (FATOR * FATOR)) div FATOR;
	seg := seg mod FATOR;
	
	writeln('Os segundos convertidos ficam: ', horas, ':', min, ':', seg);
 
End.